src/ui/LessonCard/LessonCard.tsx

```diff
@@ -0,0 +1,170 @@
+import { FC } from "react";
+import styled from "styled-components";
+import shapesBgLesson1Url from "./shapes-bg-lesson1.svg";
+import shapesBgLesson2Url from "./shapes-bg-lesson2.svg";
+import shapesBgLesson3Url from "./shapes-bg-lesson3.svg";
+import shapesBgLesson4Url from "./shapes-bg-lesson4.svg";
+import shapesBgLesson5Url from "./shapes-bg-lesson5.svg";
+import shapesMascotLesson1Url from "./shapes-mascot-lesson1.svg";
+import shapesMascotLesson2Url from "./shapes-mascot-lesson2.svg";
+import shapesMascotLesson3Url from "./shapes-mascot-lesson3.svg";
+import shapesMascotLesson4Url from "./shapes-mascot-lesson4.svg";
+import shapesMascotLesson5Url from "./shapes-mascot-lesson5.svg";
+import { H3, Subheading2, P } from "../Typography";
+import { Button } from "../Button";
+
+type MascotType =
+  | "lesson-1"
+  | "lesson-2"
+  | "lesson-3"
+  | "lesson-4"
+  | "lesson-5";
+
+const Wrap = styled.div`
+  display: flex;
+  flex-wrap: nowrap;
+  flex-direction: row;
+`;
+
+const BlockMascot = styled.div<{
+  $mascotType: MascotType;
+}>`
+  flex-grow: 2;
+  flex-basis: 0;
+  display: flex;
+  flex-wrap: nowrap;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  background-repeat: no-repeat;
+  background-position: center;
+  background-size: cover;
+  background-color: var(--color-blue-dark);
+
+  ${(props) =>
+    props.$mascotType === "lesson-1" &&
+    `
+    background-image: url(${shapesBgLesson1Url});
```

Comment:
I think instead of having a `mascotType` and transplate it to a image url, it might be more straightforward to have image url as an input prop directly

```tsx
background-image: ${props => `url(${bgUrl})`};

```

So basically it's controlled from the parent component.